styles.css

```diff
@@ -115,6 +116,16 @@ body {
   margin-top: 114px;
 }
 
+.detail-popup button {
+  background-color: white;
+  cursor: pointer;
+  border-radius: 8px;
+  margin-right: 10px;
+  border: 1px solid #6070ff;
+  width: 135px;
+  height: 48px;
+}
+
```

Comment:
Nice job implementing the functionalities of the popup👍 But your popup doesn't match the design🥲 Kindly take note of the comparison below and ensure your popup matches the design😉 add every element as seen in the design👍  This applies too for your mobile screen. do well to ensure your popup matches the design both on mobile and on desktop as it is required for this milestone👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="500" alt="yours" src="https://user-images.githubusercontent.com/64643995/233066427-e74999d7-073d-4d51-b4a9-3d51aa5109be.png">|<img width="500" alt="design" src="https://user-images.githubusercontent.com/64643995/233066579-d27b63bb-2630-4eaa-b069-0165d3c46832.png">|

Reply:
- [ ] As requested by the previeous reviewer, note the `background-color` of the space around the popup windows on Figma design, but it is not showing on your design. Kindly make sure your popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins).

### Your design :point_down: 
![](https://user-images.githubusercontent.com/64643995/233066427-e74999d7-073d-4d51-b4a9-3d51aa5109be.png)

### Figma's Design :point_down: 
![](https://user-images.githubusercontent.com/64643995/233066579-d27b63bb-2630-4eaa-b069-0165d3c46832.png)